charts/sequencer/values.yaml

```diff
@@ -38,14 +38,14 @@ config:
     # Note large balances must be strings support templating with the u128 size account balances
     genesisAccounts:
     - address: 1c0c490f1b5528d8173c5de46d131160e4b2c0c3
-      balance: "340282366920938463463374607431768211455"
+      balance: "333333333333333333"
```

Comment:
What is the reason for update here?

Reply:
fees are now being redirected to fee recipient, causing an overflow on fee recipient balance account. Took the value used in dusk-7.